doc/source/io.rst

```diff
@@ -2192,6 +2192,18 @@ indices to be parsed.
 
    read_excel('path_to_file.xls', 'Sheet1', parse_cols=[0, 2, 3])
 
+
+Parsing Dates
++++++++++++++
+
+The `parse_dates` keyword for `read_excel` is used to specify whether to parse strings
+to a datetime.
```

Comment:
Maybe note specifically that you do *no* need to use this when the data are formatted as dates in the excel file.